includes/Checker/Abstract_Check_Runner.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Abstract_Check_runner
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker;
+
+use WordPress\Plugin_Check\Checker\Check_Runner;
+use WordPress\Plugin_Check\Checker\Check_Result;
```

Comment:
No need to import these as they are in the same namespace. It's also okay to have them here, but either way we should be consistent about it. For example, you're not including the `Checks` class here, which is actually just as relevant since it is referred to in a doc-block.

So either we should remove those `use` statements and always only add those that are from another namespace, or we should include _all_ of them.